Triggers/TimeTrigger.cs

```diff
@@ -0,0 +1,30 @@
+using System.Threading.Tasks;
+using Atea.Interfaces;
+using Microsoft.Azure.WebJobs;
+
+namespace Atea
+{
+    public class TimeTrigger
+    {
+        public readonly IAddTableData _tabledata;
+        public readonly IAddBlobs _blobs;
+        public readonly IImportConnection _connection;
+
+        public TimeTrigger( IAddTableData addTableData, IAddBlobs addBlobs, IImportConnection connection)
+        {
+            _tabledata = addTableData;
+            _blobs = addBlobs;
+            _connection = connection;
+        }
+
+        [FunctionName("FetchData")]
+
+        public async Task RunTimer([TimerTrigger("0 */1 * * * *")] TimerInfo myTimer, ExecutionContext ec)
+        {
+            var importedData = _connection.DataImport();
+            var success = myTimer.Schedule != null;
```

Comment:
šis starpcitu nekad nebūs null manuprāt.

Reply:
Tev success/failure ir principā atkarīgs no tā vai pieprasījums pēc datiem bija veiksmīgs.